docs/error-messages/compiler-errors-1/compiler-error-c2394.md

```diff
@@ -1,14 +1,13 @@
 ---
-description: "Learn more about: Compiler Error C2394"
 title: "Compiler Error C2394"
+description: "Learn more about: Compiler Error C2394"
 ms.date: "11/04/2016"
 f1_keywords: ["C2394"]
 helpviewer_keywords: ["C2394"]
-ms.assetid: 653fa9a0-29b3-48aa-bc01-82f98f717a2b
 ---
 # Compiler Error C2394
 
-'your_type::operator'op'" : CLR or WinRToperator not valid. At least one parameter must be of the following types: 'T^', 'T^%', 'T^&', where T = 'your_type'
+"type::operator 'operator'": CLR/WinRT operator not valid. At least one parameter must be of the following types: 'T^', 'T^%', 'T^&', where T = 'type'
```

Comment:
Referred to the following line to tidy up the error message above:

https://github.com/MicrosoftDocs/cpp-docs/blob/3479df23bc25119d3cac5fa1efb541e19291b1a6/docs/error-messages/compiler-errors-1/compiler-errors-c2300-through-c2399.md?plain=1#L112

Reply:
Our style for these isn't consistent. I don't think I like the double quotes since they don't appear in the actual error. I tweaked this to match what you refer to, as that seems closest to what the compiler generates. This is a verbatim of what I get when I compile the error sample at the command line: `test.cpp(2): error C2394: 'Y::operator -': CLR operator not valid. At least one parameter must be of the following types: 'T^', 'T^%', 'T^&', where T = 'Y'`